CHANGELOG.md

```diff
@@ -4,6 +4,36 @@ All notable changes to the "mongodb" extension will be documented in this file.
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
-## [Unreleased]
+## [0.0.3] - 2020-05-26
 
-- Initial release
\ No newline at end of file
+### Added
+
+- `require` support in playgrounds.
+
+### Fixed
+
+- Fixed an issue with file pathing on SSL and SSH connections on windows.
+- Fixed an issue with connecting to mongodb through the shell while using SSH.
+- Fixed a connectivity issue with playgrounds and SSH connections.
+- Fixed an issue with having a babel config in a workspace root.
+
+### Changed
+
+- README copy tweaks.
+
+## [0.0.2] - 2020-05-13
+
+### Changed
+
+- Marketplace preview release
```

Comment:
Do we do dots at the end? Some release notes with and some without.

Reply:
Any preference? I feel sometimes it looks better with sometimes without lol

Reply:
I prefer when it is consistent, all with or all without :)

Reply:
Sound good - just pushed up all without